pkg/proxy/router/router.go

```diff
@@ -670,9 +673,9 @@ func NewServer(conf *Conf) *Server {
 		counter:       stats.NewCounters("router"),
 		lastActionSeq: -1,
 		startAt:       time.Now(),
-		moper:         NewMultiOperator(conf.Addr),
+		moper:         newMultiOperator(conf.Addr),
 		reqCh:         make(chan *PipelineRequest, 1000),
-		pools:         cachepool.NewCachePool(),
+		pools:         redisconn.NewPools(16, f),
```

Comment:
same here. and a few other places... probably you want to make it happen in a follow-up pr. 

I think it is really important to make the const chose wisely and gives the reason behind it.